src/smart-components/workspaces/WorkspaceListTable.tsx

```diff
@@ -198,6 +254,16 @@ const WorkspaceListTable = () => {
     dispatch(fetchWorkspaces());
   }, [dispatch]);
 
+  useEffect(() => {
+    chrome.getUserPermissions().then((permissions) => {
+      setUserPermissions(
+        permissions.find((obj) => {
+          return obj.permission === 'inventory:groups:write';
+        }),
```

Comment:
Let's add `inventory:groups:*` in here as well.
```suggestion
        permissions.find(({ permission }) => ['inventory:groups:write', 'inventory:groups:*'].includes(permission)),
```